pandas/indexes/multi.py

```diff
@@ -769,7 +771,7 @@ def to_hierarchical(self, n_repeat, n_shuffle=1):
         levels = self.levels
         labels = [np.repeat(x, n_repeat) for x in self.labels]
         # Assumes that each label is divisible by n_shuffle
-        labels = [x.reshape(n_shuffle, -1).ravel(1) for x in labels]
+        labels = [x.reshape(n_shuffle, -1).ravel(order='F') for x in labels]
```

Comment:
Yes, it does.  I worked on this method on the `numpy` end, where we deprecated non-valid arguments like `1`.  You're supposed to only pass in `C`, `F`, `A`, or `K`.